File: app/src/main/java/com/tetalab/androidtutorial_astronomy/model/AstronomyResponse.java
package com.tetalab.androidtutorial_astronomy.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AstronomyResponse {

    @SerializedName("astronomy")
    @Expose
    private Astronomy astronomy;
    @SerializedName("feedCreation")
    @Expose
    private String feedCreation;
    @SerializedName("metric")
    @Expose
    private Boolean metric;

    public Astronomy getAstronomy() {
        return astronomy;
    }

    public void setAstronomy(Astronomy astronomy) {
        this.astronomy = astronomy;
    }

    public String getFeedCreation() {
        return feedCreation;
    }

    public void setFeedCreation(String feedCreation) {
        this.feedCreation = feedCreation;
    }

    public Boolean getMetric() {
        return metric;
    }

    public void setMetric(Boolean metric) {
        this.metric = metric;
    }

}

File: app/src/main/java/com/tetalab/androidtutorial_astronomy/model/AstronomyResponse.kt
package com.tetalab.androidtutorial_astronomy.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AstronomyResponse {

    @SerializedName("astronomy")
    @Expose
    var astronomy: Astronomy? = null
    @SerializedName("feedCreation")
    @Expose
    var feedCreation: String? = null
    @SerializedName("metric")
    @Expose
    var metric: Boolean? = null

}